### PR TITLE
Upgrade Rswag to 2.13

### PR DESCRIPTION
**Pull request by stevenjcumming**
## Summary

- Upgrade `rswag-specs` and `rswag-ui ` to 2.13 
- Remove `basePath` from RswagConfig files
- Update swagger_* methods in `swagger_helper.rb` (spec/swagger_helper.rb) to use OAS 3 compatible methods
- Update rswag related documentation
- Ran `rake rswag:appeals_api:all` (no file changes) and `rake rswag:claims_api:run` (2 file changes)

## Related issue(s)
- [Issues #72416](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72416)

## Testing done
- Local testing (all swagger related examples passed)
- git diff comparison between files 
- swagger-ui visual comparison (Chrome and Safari)
- tested OAS 3 compliance with [LOAST](https://github.com/department-of-veterans-affairs/lighthouse-oas-tests)

## Screenshots
Screenshot 2023-12-21 at 8 26 02 AM

## What areas of the site does it impact?
ClaimsApi and AppealsApi api docs (swagger) 

## Acceptance Criteria
- [x] `rswag-specs` and `rswag-ui ` is  v2.13 
- [x] AppealsApi and ClaimsApi are accessible/working locally via swagger-ui
- [x] Resolve rswag deprecation warnings in CI checks
- [x] *swagger files are OAS 3 compliant 

*using LOAST the swagger files aren't compliant, _however_ the docs on [https://developer.va.gov/explore/api/benefits-claims/docs?version=current](https://developer.va.gov/explore/api/benefits-claims/docs?version=current) are NOT complaint either, so I've concluded that non-compliance doesn't mean broken. Compliance should be a separate PR with the appropriate team(s).

### CODE REVIEW

**Review by rmtolmach — APPROVED** [review]

**Comment by rmtolmach** [comment]
> Waiting on code owner review from department-of-veterans-affairs/lighthouse-banana-peels and/or department-of-veterans-affairs/lighthouse-dash

@stevenjcumming for lighthouse-banana-peels, reach out in Slack in #appeals-api. And for lighthouse-dash, #team-dash

File: modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json [hunk 1]
```diff
@@ -5806,7 +5806,7 @@
                       "status": "404",
                       "detail": "Resource not found",
                       "source": {
-                        "pointer": "/Users/rockwellrice/Documents/oddball/vets-api/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb:74:in `attachments'"
+                        "pointer": "/Users/steven.cumming/workspace/va.gov/vets-api/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb:74:in `attachments'"
                       }
```
Comment (rockwellwindsor-va) [inline modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json:5810]: There was a recent oversight (by me) on an update to our error reporting for this error which did not properly shorten this path here.  The only fix ATM is to manually shorten the path, `/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb:74:in `attachments'"` would probably be just right.  There is one other place this occurs which is on line 5809.

**Review by caseywilliams — DISMISSED** [review]
Thanks, I checked this out and confirmed that things are looking good from the banana peels side - LGTM!

**Review by rockwellwindsor-va — APPROVED** [review]
Looks good to us on team Dash

**Review by ericboehs — DISMISSED** [review]

**Review by ericboehs — DISMISSED** [review]

**Review by ericboehs — APPROVED** [review]

**Pull request merged**